spras/rwr.py

```diff
@@ -0,0 +1,200 @@
+import warnings
+from pathlib import Path
+
+import pandas as pd
+
+from spras.containers import prepare_volume, run_container
+from spras.interactome import (
+    convert_undirected_to_directed,
+    reinsert_direction_col_directed,
+)
+from spras.prm import PRM
+from spras.util import add_rank_column
+
+__all__ = ['RWR']
+
+"""
+RWR will construct a directed graph from the provided input file
```

Comment:
It's good to note that RWR assumes a directed graph. "From the provided input file" is vague, especially if the user can input a network file and a node prizes file.